src/main/scala/io/iohk/ethereum/blockchain/sync/RegularSync.scala

```diff
@@ -112,6 +127,57 @@ class RegularSync(
       }
   }
 
+  /**
+    * Handles NewHashesMessege, should only cover this message when we are top of the chain
+    */
+  def handleNewBlockHashesMessages: Receive = {
```

Comment:
`NewBlockHashes` is a broadcast message, isn't it? So I'd say this should go into `handleBroadcastedBlockMessages` or `handleBroadcastedBlockMessages` should be renamed (eg. `handleNewBlockMessages`).

Reply:
valid point, I would go with renaming ` handleBroadcastedBlockMessages` to `handleNewBlockMessages`